cli/src/commands/bonds.rs

```diff
@@ -0,0 +1,57 @@
+use crate::commands::common::CommonCollectOptions;
+use crate::{dto::ValidatorBondRecord, utils::rpc::get_rpc_client};
+use anchor_lang::AccountDeserialize;
+use serde_yaml;
+use solana_account_decoder::UiAccountEncoding;
+use solana_client::{
+    rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig},
+    rpc_filter::Memcmp,
+};
+use validator_bonds::state::bond::Bond;
+
+const BONDS_DISCRIMINATOR: [u8; 8] = [224, 128, 48, 251, 182, 246, 111, 196];
+const BONDS_PROGRAM_ID: &str = "vBoNdEvzMrSai7is21XgVYik65mqtaKXuSdMBJ1xkW4";
```

Comment:
probably not necessary while possible to load the program id from `validator-bonds` when it's as a dependency
https://github.com/marinade-finance/validator-bonds/blob/main/programs/validator-bonds/src/constants.rs#L4

the reason why it's duplicated in contract code is that there is (at least at anchor 0.29.0 what I believe) a way to get the program id to anchor IDL. So there is already duplication of the program ids in contract as it's in `lib.rs` as well
https://github.com/marinade-finance/validator-bonds/blob/main/programs/validator-bonds/src/lib.rs#L31
but I added at least a cargo test to get it failed when something changes